docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -19,13 +19,57 @@ artifacts of the corresponding domain.
 
 #### Prerequisites
 
-Before you begin, we recommend the following:
+Before you begin, you need to perform the steps below:
 
 * Review the [Domain resource]({{< relref "/userguide/managing-domains/domain-resource/_index.md" >}}) documentation.
-* Review the [operator prerequisites](https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/#operator-prerequisites)
-section for the supported versions of Kubernetes and Helm.
-* Complete the preliminary required steps documented [here]({{< relref "/userguide/managing-fmw-domains/soa-suite/_index.md" >}}).
+* Review the [system requirements]({{< relref "/userguide/managing-fmw-domains/soa-suite#prerequisites-for-soa-suite-domains" >}}) for deploying Oracle SOA Suite domains on Kubernetes.
+* Make sure that Kubernetes is set up in the environment. For details, see the [Kubernetes setup guide]({{< relref "/userguide/overview/k8s-setup.md" >}}).
+* Make sure that the WebLogic operator is running. See [Manage operators]({{< relref "/userguide/managing-operators/_index.md" >}}) for operator infrastructure setup and [Install the operator]({{< relref "/userguide/managing-operators/installation/_index.md" >}}) for operator installation. Make sure you install the operator of version 2.4.0.
```

Comment:
WebLogic operator -> WebLogic Kubernetes Operator
operator of version 2.4.0. -> operator version 2.4.0.